README.md

```diff
@@ -14,33 +16,46 @@
 - CanCanCan
 - Bullet
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo Link](https://git.heroku.com/rocky-harbor-76582.git)
```

Comment:
Thank you for your observation, I mistakenly uploaded the Heroku repository link instead of the deployed link. The correct [link](https://rocky-harbor-76582.herokuapp.com) is already on the Readme and working as expected.